### PR TITLE
Exports the toggle all notes function

### PR DESCRIPTION
**Pull request by SHession**
Exports the toggle all notes function, so that it can be used for shortcuts in Composer.

### CODE REVIEW

File: pages/dist/styles.css [hunk 1]
```diff
@@ -5,8 +5,11 @@
 .ProseMirror {
   word-wrap: break-word;
   white-space: pre-wrap;
+  white-space: break-spaces;
```
Comment (jonathonherbert) [inline pages/dist/styles.css:8]: What's the reason for the CSS change?

Comment (SHession) [reply pages/dist/styles.css:8]: Honestly, not sure. This was just added when I ran "npm run build", I'm not sure why it would have done that.

Comment (jonathonherbert) [reply pages/dist/styles.css:8]: Hm, I guess there was something latent in the styling that hadn't been checked in. It shouldn't affect the releases, so no worries!

Comment (SHession) [reply pages/dist/styles.css]: I reset those changes just to be safe. Possibly it should be updated to include them at some point...

**Pull request merged**